### PR TITLE
Update github URLs and mailing list URLs

### PR DESCRIPTION
**Pull request by tas50**
Also auto fixed the MD format in the contributing doc and fixed a bunch
of spelling mistakes there.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,30 +19,29 @@ You can [become a contributor by signing the ICLA or by contributing on behalf o
...
+- [Chef Docs](https://docs.chef.io/)
+- [Learn Chef](https://learn.chef.io/)
+- Chef [product page](https://www.chef.io/chef)
 
-Overview
---------
+## Overview
 If you're experienced with the toolchain, here are the steps for submitting a patch to Supermarket:
-
 1. [Fork the project](https://github.com/chef/supermarket/fork) on GitHub
```
Comment (robbkidd) [inline CONTRIBUTING.md]: The mixed `ol` and `ul` list items here seem to confuse Markdown. I suggest replacing the numbers with dashes and turn the whole thing into `ul`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -59,32 +52,18 @@ Good: Add some feature
 ```
 
 ### Writing Tests
-In order to ensure the integrity of the project (and prevent regressions), we _cannot_ merge any patch that does not have adequate test coverage. Even if you have never written tests before, the existing tests serve as great boilerplate examples. At minimum, changes to a model must have a unit spec, changes to a controller must have a request spec, changes to a view must have a view or capybara spec, changes to the javascript must have a polgergist spec.
+In order to ensure the integrity of the project (and prevent regressions), we _cannot_ merge any patch that does not have adequate test coverage. Even if you have never written tests before, the existing tests serve as great boilerplate examples. At minimum, changes to a model must have a unit spec, changes to a controller must have a request spec, changes to a view must have a view or capybara spec, changes to the javascript must have a poltergeist spec.
```
Comment (robbkidd) [inline CONTRIBUTING.md]: "polgergist" was from the Scandinavian word for "awesome, super-fly code snippet" but your change is acceptable, too.

**Comment by tas50** [comment]
@robbkidd updated the markdown

**Comment by robbkidd** [comment]
[Markdown render has a sad](https://github.com/tas50/supermarket/blob/c6b61383fc7deb5c0828a80a631cc00af27d14fb/CONTRIBUTING.md#overview). I think an empty line between the "here are the steps" intro line and the list will help. And maybe turning the numbered list into bullets so that numbering does not need know it ought to resume after the code block.

**Comment by tas50** [comment]
*sigh*  Looked fine on my box in Atom

**Comment by tas50** [comment]
I just edited in the GUI.  Github does some strange things with markdown that other apps don't.

**Comment by robbkidd** [comment]
:+1:

**Comment by robbkidd** [comment]
:+1:

**Pull request merged**